netrd/reconstruction/template.py

```diff
@@ -0,0 +1,72 @@
+"""
+<file_name>.py
+
+Graph reconstruction algorithm based on <link to paper, or website, or repository>.
+
+author: <your name here>
+email: <name at server dot com> (optional)
+Submitted as part of the 2019 NetSI Collabathon.
+"""
+
+from .base import BaseReconstructor
+
+class <AlgorithmName>(BaseReconstructor):
+    def fit(self, T):
+        """A brief one-line description of the algorithm goes here.
+
+        A short paragraph may follow. The paragraph may include $latex$ by
+        enclosing it in dollar signs $\textbf{like this}$.
+
+        Params
+        ------
+        T (np.ndarray): Array consisting of $T$ observations from $N$ sensors.
```

Comment:
Can you please add that the array N rows and T columns? Also, as written right now, T stands for both the variable containing the time series data as well as the number of columns.

Reply:
Yeah, I don't love the double use of $T$. Was thinking that we could use $t$ for the number of observations, so $T$ is a $t \times N$ matrix.

Reply:
That change should probably be a separate PR, since we'll have to touch the docs on the other reconstruction files.

Reply:
@jkbren do you agree with the use of T and t? I generally avoid single character names like the plague, but since we are trying to standardize things a bit, it might be a good idea to have G and T mean one and the same thing throughout the package.
Another option is TS and T.

Reply:
@sdmccabe Agreed, that's another PR.